### ISSUE TITLE
[R6] Error on server stop 

### ISSUE THREAD

**Issue by xenos5077** [VolmitSoftware/React#141]
On server stopping this error does appear.
Spigot 1.12
React-6.26411a-origin

```[15:40:38 ERROR]: Error occurred while disabling React v6.26411a (Is it up to date?)
java.lang.NoSuchMethodError: null
        at java.lang.Thread.destroy(Unknown Source) ~[?:1.8.0_151]
        at react.surge.Main.onStop(Main.java:242) ~[?:?]
        at react.surge.control.AmpedPlugin.onDisable(AmpedPlugin.java:90) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:361) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:421) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:414) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.disablePlugins(CraftServer.java:342) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at net.minecraft.server.v1_12_R1.MinecraftServer.stop(MinecraftServer.java:464) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:612) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_151]```

Better visibility here : https://hastebin.com/omusufureq.css

**Comment by cyberpwnn** [comment]
You can use three ` instead :P

**Comment by cyberpwnn** [comment]
Fixed in 6.27159a
